modules/virt-running-real-time-checkup.adoc

```diff
@@ -0,0 +1,214 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-running-cluster-checkups.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-running-real-time-checkup_{context}"]
+= Running a real-time checkup
+
+Use a predefined checkup to verify that your {product-title} cluster can run a virtualized real-time workloads.
+
+You run a real-time checkup by performing the following steps:
+
+. Create a service account, role, and role bindings for the real-time checkup.
+. Create a config map to provide the input to run the checkup and to store the results.
+. Create a job to run the checkup.
+. Review the results in the config map.
+. Optional: To rerun the checkup, delete the existing config map and job and then create a new config map and job.
```

Comment:
:s/ and then create a new config map and job/ and then recreate the config map and the job
If this option discusses re-running the checkup, then it should be reflected that the same ConfigMap and Job resources can and should be recreated.

Reply:
I have removed these "summary" steps from the content. I don't think they are necessary.